src/components/settings-menu/UserDevSection.js

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from 'react';
+import { InteractionManager } from 'react-native';
+import { ScrollView, Switch } from 'react-native-gesture-handler';
+import { useDispatch } from 'react-redux';
+import networkTypes from '../../helpers/networkTypes';
+import { settingsUpdateNetwork } from '../../redux/settings';
+import { Column } from '../layout';
+import { ListFooter, ListItem } from '../list';
+import NetworkSection from './NetworkSection';
+
+import {
+  useAccountSettings,
+  useInitializeAccountData,
+  useLoadAccountData,
+  useResetAccountState,
+} from '@rainbow-me/hooks';
+const UserDevSection = () => {
+  const dispatch = useDispatch();
+
+  const {
+    testnetsEnabled,
+    settingsChangeTestnetsEnabled,
+  } = useAccountSettings();
+
+  const resetAccountState = useResetAccountState();
+  const loadAccountData = useLoadAccountData();
+  const initializeAccountData = useInitializeAccountData();
+
+  const revertToMainnet = useCallback(async () => {
+    await resetAccountState();
```

Comment:
hey other than updating the network, is there any reason we need to reset the account / reload account data / reinit account data?

Reply:
i'm following what we're doing in `NetworkSection.js`